### ISSUE TITLE
Rework how mutators display

### ISSUE THREAD

**Issue by darklordabc** [darklordabc/Legends-of-Dota-Redux#233]
- Some Mutators should be able to to be clicked multiple times to cycle through settings
  e.g. the "Strong Creeps" mutator become "Stronger Creeps" and then "Extreme Creeps", and then to "Normal Creeps" (which is disabled). There should be some graphical representation to show which cycle you are in, maybe a different icon but that will require a lot of work. 
- Some Mutators should change their text. For example, max level should be either "Max Level 25" to "Max Level 50" to "Max Level 100"
- More complicated mutators like fast start, which are based on more than 1 property, should change cycle/text based on a range of values. For example respawn, if respawn time is less than 100 but greater than 50, it should be "fast respawn" if its less than 50 and greater than 25, should be "Faster respawn", if respawn is less than 25% it should be "Insta Respawn" (or something).  Mutators like fast start should work a similar way.  

Basically I want mutators to more accurately represent the settings of the game, so that players can look at them and immediately see the settings of the game. Right now they are not flexible enough and will become disabled if their predefined settings are not exactly matched.
